client/src/features/calendar/MonthAndYear.js

```diff
@@ -42,7 +42,7 @@ const MonthAndYear = ({
           </svg>
         </button>
       </div>
-      <h2 className="ml-2 w-[16ch] text-3xl font-bold leading-none">
+      <h2 className="ml-2 w-[12ch] text-3xl font-bold leading-none">
```

Comment:
This change seems to force September to create a new line. Changing this to 14 seems to correct the issue.

Reply:
@Natashka333 this should satisfy your feedback.

Reply:
thank you for finding that - have resolved that in commit [6e88062](https://github.com/Together-100Devs/Together/pull/388/commits/6e88062d453ef1b54be70f6191096d15481a7013)